core/src/main/scala/org/http4s/TransferCoding.scala

```diff
@@ -18,23 +18,52 @@
  */
 package org.http4s
 
-import org.http4s.syntax.string._
+import cats.{Order, Show}
+import cats.data.NonEmptyList
 import org.http4s.util._
+import org.http4s.parser.{Http4sParser, Rfc2616BasicRules}
 
-final case class TransferCoding private (coding: CaseInsensitiveString) extends Renderable {
+sealed abstract case class TransferCoding private (coding: String) extends Renderable {
```

Comment:
This could also extend `Ordered` for better non-Cats Scala integration.

I've been phasing out `Renderable` in favor of a `Renderer`: ad hoc polymorphism over inheritance.

Reply:
This is used in a `Header` which leans to `Renderable` classes. However it is a good goal, I’ll try to use `Renderer` in the header